src/main/kotlin/com/demonwav/mcdev/util/McPsiUtil.kt

```diff
@@ -37,6 +39,7 @@ import org.jetbrains.annotations.Contract
 import java.util.stream.Stream
 
 // Parent
+fun PsiElement.findModule(): Module? = ProjectRootManager.getInstance(project).fileIndex.getModuleForFile(containingFile.virtualFile)
```

Comment:
There is `ModuleUtilCore.findModuleForPsiElement` which will probably handle more cases than this does